### PR TITLE
[SDPAP-7103] Enable bulk publishing for Approvers

### PR DESCRIPTION
**Pull request by vincent-gao**
### Jira
https://digital-vic.atlassian.net/browse/SDPAP-7103

### Change
The update includes the implementation of the `tide node bulk update` permission and the necessary code changes.

### Related PR
- https://github.com/dpc-sdp/dev-tools/pull/104

### CODE REVIEW

File: dev-tools.sh [hunk 1]
```diff
@@ -33,6 +33,6 @@
 # stable version.
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
-# export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=4522e05d991e675aa80dfbc6e7a471896f3f6d81
```
Comment (krakerag) [inline dev-tools.sh:36]: Do we need this reverted?

File: tests/behat/features/cms_support.feature [hunk 2]
```diff
@@ -1,11 +1,11 @@
 @tide @install:toolbar
-Feature: CMS Support
+feature: cms support
 
   @api
-  Scenario Outline: User logs into the CMS and clicks on the Manage tab
-    Given I am logged in as a user with the "<role>" role
-    Then I should find menu item text matching "CMS Support"
-    Examples:
+  scenario outline: user logs into the cms and clicks on the manage tab
```
Comment (krakerag) [inline tests/behat/features/cms_support.feature]: Did some filter go over this to make it all lowercase? Looks weird :)

Comment (vincent-gao) [reply tests/behat/features/cms_support.feature]: yeah, not sure why seems like an accidental commit. deleted.

File: dev-tools.sh (hunk 1 above)
Comment (vincent-gao) [reply dev-tools.sh:36]: yes, it will be reverted.

**Comment by vincent-gao** [comment]
hey @krakerag , the dev-tools change has been reverted. I think the PR is now ready to be merged.

**Review by krakerag — APPROVED** [review]

**Pull request merged**